cmd/appliance/upgrade/complete.go

```diff
@@ -417,36 +417,55 @@ func upgradeCompleteRun(cmd *cobra.Command, args []string, opts *upgradeComplete
 		}
 		for _, appliance := range appliances {
 			i := appliance
+			logger := log.WithFields(log.Fields{
+				"appliance": i.GetName(),
+				"id":        i.GetId(),
+			})
 			var initialVolume float32
 			for _, appData := range initialStats.GetData() {
 				if i.GetId() == appData.GetId() {
 					initialVolume = appData.GetVolumeNumber()
 				}
 			}
+			logger.WithField("volume", initialVolume).Info("registered initial volume number")
 			g.Go(func() error {
 				ctx, cancel := context.WithTimeout(ctx, opts.Timeout)
 				defer cancel()
-				logEntry := log.WithField("appliance", i.GetName())
-				logEntry.Info("checking if ready")
+				logger := log.WithFields(log.Fields{
+					"appliance": i.GetName(),
+					"id":        i.GetId(),
+				})
```

Comment:
Can we reuse the same logger declared in line 420?

Reply:
yes we can :+1: